test/tests/run/run_target.vader

```diff
@@ -133,7 +133,7 @@ Expect ([CMakeRun] See '123 F1' result in quickfix):
   || 123 F1
 
 Execute (Run test_app target with mask arguments):
-    if !has('win32') || utils#cmake#verNewerOrEq([3, 14])
+    if !has('win32') && utils#cmake#verNewerOrEq([3, 14])
```

Comment:
The same comment as below. This test works in the master branch for all cmake versions.